lib/__tests__/index.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const assert = require('assert');
```

Comment:
I think we can get rid of using `assert` and use the assertions that jest provides via [expect](https://facebook.github.io/jest/docs/en/expect.html#content)